components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -398,9 +399,8 @@ public void addAPI(API api) throws APIManagementException {
                     contextCache.put(api.getContext(), true);
                 }
             }
-        } catch (APIManagementException e) {
-            throw new APIManagementException("Error in adding API - " + api.getId().getApiName() + "-" +
-                                             api.getId().getVersion() + ". " + e.getMessage(), e);
+        } catch (APIManagementException e) {          
```

Comment:
Catching and throwing the same exception. Any reason for doing that? Is it to change the error message?